src/app/profile/yourGiving/givingRecipientView/givingRecipientView.tpl.html

```diff
@@ -0,0 +1,2 @@
+<donation-recipient ng-repeat="recipient in $ctrl.recipients | orderBy:'designation-name'"
```

Comment:
I'm betting this is old data as well as "recent" being defined in the API as last 2 years.